extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -57,6 +58,7 @@ export const buildDacpacNotFound = localize('buildDacpacNotFound', "Dacpac creat
 export function projectAlreadyOpened(path: string) { return localize('projectAlreadyOpened', "Project '{0}' is already opened.", path); }
 export function projectAlreadyExists(name: string, path: string) { return localize('projectAlreadyExists', "A project named {0} already exists in {1}.", name, path); }
 export function mssqlNotFound(mssqlConfigDir: string) { return localize('mssqlNotFound', "Could not get mssql extension's install location at {0}", mssqlConfigDir); }
+export function projBuildFailed(errorMessage: string) { return localize('projBuildFailed', "Build failed.  Check output pane for more details.  {0}}", errorMessage); }
```

Comment:
there are 2 places in the resource string that are using double space, also there is an extra } at the end.

Reply:
I was always taught two spaces between sentences?  But I'll begrudgingly change it to one since that appears to be how the rest of ADS is. :)